includes/services/extended/yahoo.php

```diff
@@ -69,7 +70,7 @@ function build_token_meta( $token ) {
 		return apply_filters( 'keyring_access_token_meta', $meta, $this->get_name(), $token, $response, $this );
 	}
 
-	function get_display( Keyring_Access_Token$token ) {
+	function get_display( Keyring_Access_Token $token ) {
```

Comment:
I'm assuming this doesn't change anything, but I've never seen anything like `Keyring_Access_Token$token` before. I'd have expected it to error.

Reply:
PHP is [wild like that](https://3v4l.org/0EhpT). 🙃